### PR TITLE
Fix handlebars and paused state

### PR DESCRIPTION
**Pull request by roggervalf**
#### Changes Made
This PR fixes handlebars vulnerability #390 and I noticed that from bull, when entering to paused page is getting an error, so I fixed that issue too, as bullmq has paused state too, I also added it to it.
#### Potential Risks
Now we are allowed to enter to paused state page without errors

#### Test Plan
1.Go to example directory
2.Run: npm run start:bull
3.Enter to the first queue
4.Click in pause button
5.Enter to paused state, you should enter to that page without error, and for those active job should be in paused state
4.Same case for bullmq

This PR is taking advantage of #375 
#### Checklist

- [ ] I've increased test coverage
- [x] Since this is a public repository, I've checked I'm not publishing private data in the code, commit comments, or this PR.

### CODE REVIEW

**Comment by roggervalf** [comment]
@bradvogel please when you get some change for some :eyes: on this pr

File: public/dashboard.js [hunk 1]
```diff
@@ -227,4 +227,59 @@ $(document).ready(() => {
...
   });
+
+  $('.js-pause-queue').on('click', function (e) {
+    e.preventDefault();
+    $(this).prop('disabled', true);
+    const queueName = $(this).data('queue-name');
+    const queueHost = $(this).data('queue-host');
+
+    const response = window.confirm(
+      `Do you really want to pause the queue "${queueHost}/${queueName}"?`
```
Comment (bradvogel) [inline public/dashboard.js:238]:
```suggestion
      `Are you sure you want to pause the queue "${queueHost}/${queueName}"?`
```

File: src/server/views/helpers/queueHelpers.js [hunk 2]
```diff
@@ -46,6 +46,10 @@ const Helpers = {
     return stats;
   },
 
+  isPaused: async function (queue) {
```
Comment (bradvogel) [inline src/server/views/helpers/queueHelpers.js:49]: When I run `npm run start:bee` in the examples folder and load http://localhost:4735/Queue%20Server%201/name_of_my_queue, I get the following error:
```
(node:44450) UnhandledPromiseRejectionWarning: TypeError: queue.isPaused is not a function
    at Object.isPaused (/Users/brad/dev/arena/src/server/views/helpers/queueHelpers.js:50:18)
    at handler (/Users/brad/dev/arena/src/server/views/dashboard/queueDetails.js:25:39)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

File: public/dashboard.js [hunk 3]
```diff
@@ -227,4 +227,59 @@ $(document).ready(() => {
...
+    }
+  });
+
+  $('.js-resume-queue').on('click', function (e) {
+    e.preventDefault();
+    const queueName = $(this).data('queue-name');
+    const queueHost = $(this).data('queue-host');
+
+    const response = window.confirm(
+      `Do you want to resume the queue "${queueHost}/${queueName}"?`
```
Comment (bradvogel) [inline public/dashboard.js:265]:
```suggestion
      `Are you sure you want to resume the queue "${queueHost}/${queueName}"?`
```

**Review by bradvogel — CHANGES_REQUESTED** [review]

**Comment by bradvogel** [comment]
When you're ready for me to re-review, just click this button:
image

**Comment by roggervalf** [comment]
> When you're ready for me to re-review, just click this button:
> image
Thank you @bradvogeSorry for the delay

**Comment by roggervalf** [comment]
Thank you @bradvogel, sorry for the delay

**Review by bradvogel — APPROVED** [review]
tested locally

**Pull request merged**